scripts/prepdocs.ps1

```diff
@@ -1,9 +1,9 @@
 ./scripts/loadenv.ps1
 
-$venvPythonPath = "./scripts/.venv/scripts/python.exe"
+$venvPythonPath = "./.venv/scripts/python.exe"
 if (Test-Path -Path "/usr") {
   # fallback to Linux venv path
-  $venvPythonPath = "./scripts/.venv/bin/python"
+  $venvPythonPath = "./.venv/bin/python"
 }
 
 Write-Host 'Running "prepdocs.py"'
```

Comment:
Missing path update in line 69 from `./scripts/prepdocs.py` to `./app/backend/prepdocs.py`

Reply:
i hit this issue as well

Reply:
Thanks, fixed!